packages/bpk-component-code/examples.js

```diff
@@ -44,4 +44,13 @@ const BlockAlternate = () => (
   </BpkCodeBlock>
 );
 
-export { Inline, InlineAlternate, Block, BlockAlternate };
+const MixedExample = () => (
+  <div>
+    <Inline />
+    <InlineAlternate />
+    <Block />
+    <BlockAlternate />
+  </div>
+);
+
+export { Inline, InlineAlternate, Block, BlockAlternate, MixedExample };
```

Comment:
I wonder if we should have a function called `withMixedExample` so we can do `export withMixedExample({e1, e2...})`

Something like
```
withMixedExample = examples => {
  const MixedExample = <div>{Object.keys(examples).map(Example => <Example />)}</div>;
  return {...examples, MixedExample}
}
```